support-frontend/assets/pages/contributions-landing/components/ContributionTypeTabs.jsx

```diff
@@ -94,4 +94,30 @@ function ContributionTypeTabs(props: PropTypes) {
   );
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(ContributionTypeTabs);
+function withoutProps() {
+  return (
+    <fieldset className={classNameWithModifiers('form__radio-group', ['tabs', 'contribution-type'])}>
+      <legend className={classNameWithModifiers('form__legend', ['radio-group'])}>Recurrence</legend>
+      <ul className="form__radio-group-list form__radio-group-list--border">
+        {
+          ['a','b','c'].map(id => {
```

Comment:
in fact, if we happen to know there are no AB tests running we could just use the actual one.  I think the tabs ordering test was running, which is why these were greyed out